libs/store/src/lib/payloads/effects.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  AppPlayerId,
+  AppPlayerLog,
+  AppRaceId,
+  AppTournamentId,
+} from '@razor/models';
+
+//effectsPayloads
```

Comment:
Please use `PascalCase` in types.

Reply:
Fixed in PR #14 under commit 9d9abb3b34f8680fbfaadb6ffbcdaa1043ca0d28